ios/Sources/MediaFeature/MediaListView.swift

```diff
@@ -0,0 +1,164 @@
+import ComposableArchitecture
+import Model
+import Styleguide
+import SwiftUI
+
+struct MediaListView: View {
+
+    private let store: Store<MediaListState, MediaListAction>
+    @ObservedObject private var viewStore: ViewStore<ViewState, ViewAction>
+
+    init(store: Store<MediaListState, MediaListAction>) {
+        self.store = store
+        self.viewStore = .init(store.scope(state: ViewState.init(state:), action: MediaListAction.init(action:)))
+    }
+
+    struct ViewState: Equatable {
+        var hasBlogs: Bool
+        var hasVideos: Bool
+        var hasPodcasts: Bool
+        var isSearchResultVisible: Bool
+        var isMoreActive: Bool
+
+        init(state: MediaListState) {
+            hasBlogs = !state.list.blogs.isEmpty
+            hasVideos = !state.list.videos.isEmpty
+            hasPodcasts = !state.list.podcasts.isEmpty
+            if case let .searchText(text) = state.next,
+               !text.isEmpty {
+                isSearchResultVisible = true
+            } else {
+                isSearchResultVisible = false
+            }
+            if case .more = state.next {
+                isMoreActive = true
+            } else {
+                isMoreActive = false
+            }
+        }
+    }
+
+    enum ViewAction {
+        case moreDismissed
+    }
+
+    var body: some View {
+        ZStack {
+            ScrollView {
+                VStack(spacing: 0) {
```

Comment:
It seems better to use `Group` if spacing in zero.

Reply:
It seems okay to just remove the `VStack`.